schema/rdl_schema_0.1.json

```diff
@@ -949,6 +918,61 @@
 				"TCY"
 			]
 		},
+		"Resource": {
+			"title": "Resource",
+			"type": "object",
+			"description": "Information about an individual version of the dataset.",
+			"required": [
+				"id",
+				"title",
+				"description",
+				"media_type",
```

Comment:
I don't think we want `media_type` to be required as I expect some folks might have difficulty populating it.
```suggestion

```